src/layer/tile/TileLayer.js

```diff
@@ -93,10 +93,10 @@ export var TileLayer = GridLayer.extend({
 
 			if (!options.zoomReverse) {
 				options.zoomOffset++;
-				options.maxZoom--;
+				options.maxZoom = Math.max(options.minZoom, options.maxZoom - 1);

```

Comment:
shoot; I accidentally included the fix to #7328. I will revert this @johnd0e

Reply:
For some reason my last commit isn't being added to this PR; perhaps because you've approved it @johnd0e

Reply:
> perhaps because you've approved it @johnd0e

I don't think that there can be any connection. But whatever..

Reply:
Looks like it worked! The reverts commit has been added! @johnd0e